apps/web/src/views/BuyCrypto/components/AccordianDropdown/AccordianItem.tsx

```diff
@@ -9,13 +9,33 @@ import { useTranslation } from '@pancakeswap/localization'
 import { isMobile } from 'react-device-detect'
 import formatLocaleNumber from 'utils/formatLocaleNumber'
 import { providerFeeTypes } from 'views/BuyCrypto/constants'
+import Image from 'next/image'
+import { useBuyCryptoState } from 'state/buyCrypto/hooks'
 import OnRampProviderLogo from '../OnRampProviderLogo/OnRampProviderLogo'
+import pocketWatch from '../../../../../public/images/pocket-watch.svg'
 
 const DropdownWrapper = styled.div<{ isClicked: boolean }>`
   display: ${({ isClicked }) => (isClicked ? 'none' : 'block')};
   width: 100%;
   transition: display 0.6s ease-in-out;
 `
+const targetDate = new Date('2023-08-30T00:00:00')
+
+const getTimeUntilDate = (date: Date) => {
+  const currentDate = new Date()
+  // @ts-ignore
+  const timeDifference = date - currentDate
+
+  const daysLeft = Math.floor(timeDifference / (1000 * 60 * 60 * 24))
+  const hoursLeft = Math.floor((timeDifference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60))
+  const minutesLeft = Math.floor((timeDifference % (1000 * 60 * 60)) / (1000 * 60))
+
+  return {
+    d: daysLeft,
+    h: hoursLeft,
+    m: minutesLeft,
+  }
+}
```

Comment:
U can use `getTimePeriods`
import getTimePeriods from '@pancakeswap/utils/getTimePeriods'

Reply:
ohh thanks momota, didnt know this existed. brilliant makes cleaner thank you